BepInEx.DeveloperConsole/DeveloperConsole.cs

```diff
@@ -18,17 +18,24 @@ public partial class DeveloperConsole : BaseUnityPlugin
         private static string TotalLog = "";
         private Rect UI = new Rect(20, 20, 400, 400);
         private static Vector2 scrollPosition = Vector2.zero;
+        private GUIStyle logTextStyle = new GUIStyle();
 
         public static ConfigFile BepinexConfig { get; } = new ConfigFile(Utility.CombinePaths(Paths.ConfigPath, "BepInEx.cfg"), false);
 
-        public static ConfigWrapper<int> LogDepth { get; private set; }
-        public static ConfigWrapper<bool> LogUnity { get; private set; }
+        public static ConfigEntry<int> LogDepth { get; private set; }
+        public static ConfigEntry<int> fontSize { get; private set; }
+        public static ConfigEntry<bool> LogUnity { get; private set; }
+        public static ConfigEntry<KeyboardShortcut> ToggleUIShortcut { get; private set; }
 
         public DeveloperConsole()
         {
-            LogDepth = Config.Wrap("Config", "Log buffer size", "Size of the log buffer in characters", 16300);
-            LogUnity = BepinexConfig.Wrap("Logging", "UnityLogListening", "Enables showing unity log messages in the BepInEx logging system.", true);
-                       
+            LogDepth = Config.Bind<int>("Config", "Log buffer size", 16300, "Size of the log buffer in characters.");
+            fontSize = Config.Bind<int>("Config", "Font Size", 16, new ConfigDescription("Adjusts the fontSize of the log text.", new AcceptableValueRange<int>(8, 80)));
+            LogUnity = Config.Bind<bool>("Logging", "UnityLogListening", true, "Enables showing unity log messages in the BepInEx logging system.");
+            ToggleUIShortcut = Config.Bind<KeyboardShortcut>("Config", "Toggle UI Shortcut", new KeyboardShortcut(KeyCode.Pause), "Toggles the visibility of the developer console.");

```

Comment:
You don't need to specify the generic parameter, because it is automatically inferred from the parameter.

So make it just

```cs
LogDepth = Config.Bind("Config", "Log buffer size", 16300, "Size of the log buffer in characters.");
fontSize = Config.Bind("Config", "Font Size", 16 ...
LogUnity = Config.Bind("Logging", "UnityLogListening", true, ...
ToggleUIShortcut = Config.Bind("Config", "Toggle UI Shortcut", new KeyboardShortcut ...
```